src/main/java/org/broadinstitute/hellbender/engine/spark/datasources/ReadsSparkSource.java

```diff
@@ -85,9 +82,11 @@ public ReadsSparkSource(final JavaSparkContext ctx, final ValidationStringency v
      * @param traversalParameters parameters controlling which reads to include. If <code>null</code> then all the reads (both mapped and unmapped) will be returned.
      * @param splitSize maximum bytes of bam file to read into a single partition, increasing this will result in fewer partitions. A value of zero means
      *                  use the default split size (determined by the Hadoop input format, typically the size of one HDFS block).
+     * @param useIntelInflater whether to use the Intel inflater to decompress DEFLATE streams
+     * @param useIntelDeflater whether to use the Intel deflater to compress DEFLATE streams
      * @return RDD of (SAMRecord-backed) GATKReads from the file.
      */
-    public JavaRDD<GATKRead> getParallelReads(final String readFileName, final String referencePath, final TraversalParameters traversalParameters, final long splitSize) {
+    public JavaRDD<GATKRead> getParallelReads(final String readFileName, final String referencePath, final TraversalParameters traversalParameters, final long splitSize, boolean useIntelInflater, boolean useIntelDeflater) {
```

Comment:
Actually, it would make more sense to pass the deflater/inflater settings in to the constructors for `ReadSparkSource`/`Sink` instead of on every call to `getParallelReads()`, since these settings are going to stay constant throughout a given run.